utils/api.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+export const fetchArticles = (article_id) => {
+  let endpointString = "https://nc-news-uld9.onrender.com/api/articles";
+
+  if (article_id !== undefined) {
+    endpointString += `/${article_id}`;
+  }
+  console.log(endpointString);
+  return axios
+    .get(endpointString)
+    .then(({ data }) => {
+      return data;
```

Comment:
if you are more specific with what you return here then you won't have to do res.articles in the useEffect

Reply:
Have not changed this as re-using the function to get a single article, which is returned on a .article key